common/sagas/wallet.tsx

```diff
@@ -168,7 +168,8 @@ function* broadcastTx(action: BroadcastTxRequestedAction): SagaIterator {
           txHash={txHash}
           blockExplorer={network.blockExplorer}
         />,
-        0
+        0,
+        Math.random()
```

Comment:
My aim here wasn't to replace the id here with `Math.random()`, it was to assign the id to `Math.random()` in the `showNotification` function. Otherwise, _every_ call to `showNotification` will have to remember to add a 4th arg of `Math.random()`.